### ISSUE TITLE
SyntaxError: invalid syntax - pygame 1.9.6

### ISSUE THREAD

**Issue by davidzwa** [AustL/PygameWidgets#7]
```
Traceback (most recent call last):
  File "VENV_PATH_HERE/tryouts/pygame-matplotlib.py", line 5, in <module>
    from pygame_widgets import Button
  File "VENV_PATH_HERE\lib\site-packages\pygame_widgets\__init__.py", line 2, in <module>
    from pygame_widgets.button import Button, ButtonArray
  File "VENV_PATH_HERE\lib\site-packages\pygame_widgets\button.py", line 160
    if (parent := super().get(attr)) is not None:
               ^
SyntaxError: invalid syntax
```

Maybe not python 3 compatible or not 3.7 at least?

**Comment by davidzwa** [comment]
Definitely a 3.7 issue. Might be nice to allow backwards compat. The assignment operator is not really required.

**Comment by AustL** [comment]
Fixed - see [Issue #8](https://github.com/AustL/PygameWidgets/issues/8)
